OrbitFuzzing/elfFuzzer.cpp

```diff
@@ -0,0 +1,10 @@
+#include "ElfFile.h"
+#include <cstdio>
+#include <cstdlib>
+
+
+extern "C"
+int LLVMFuzzerTestOneInput(uint8_t *buf, size_t len) {
+ ElfFile::Create(reinterpret_cast<const char*>(buf), len);
```

Comment:
you don't need this reinterpret_cast anymore.

Reply:
Right. Missed that. Thanks.